appveyor.yml

```diff
@@ -17,7 +17,12 @@ configuration:
 
 build:
   project: src/Everything.sln
-  
+
+before_test:
+  - ps: cl.exe /c /analyze /analyze:quiet- /W4 /analyze:autolog:ext sarif src\Sarif.FunctionalTests\ConverterTestData\PREfast\src\AdvancedAliasing\AA_IndexAlias.cpp; ls
```

Comment:
Is the "`ls`" at the end just so you could see in the build log that the file had been created?